pkg/storaged/details.js

```diff
@@ -1553,23 +1558,27 @@
             else
                 def_action = actions[0];  // Start
 
-            return mustache.render(mdraid_detail_tmpl,
-                                   { MDRaid: mdraid_model,
-                                     MDRaidButton: mustache.render(action_btn_tmpl,
-                                                                   { arg: mdraid.path,
-                                                                     def: def_action,
-                                                                     actions: actions
-                                                                   }),
-                                     Block: block_model,
-                                     Members: client.mdraids_members[mdraid.path].map(make_member),
-                                     DynamicMembers: (mdraid.Level != "raid0"),
-                                     Content: (block &&
-                                               mustache.render(content_tmpl,
-                                                               { Title: _("Content"),
-                                                                 path: block.path,
-                                                                 Entries: block_content_entries(block)
-                                                               }))
-                                   });
+            return [ mustache.render(mdraid_detail_tmpl,
```

Comment:
Could you name these parts? Just having an array is difficult to read if you're not familiar with the code.
Right now these aren't documented and can only be understood by reading the use: https://github.com/cockpit-project/cockpit/pull/4854/files#diff-59bbb3cde23e86514ebe995dc75261d3R1747